styles.css

```diff
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+  height: 100vh;
+  overflow: hidden;
+}
+
+h1 {
```

Comment:
You must remove h1 style to more clean Css style because you cannot use h1 tag at index.html file